### PR TITLE
WIP: Monorepo fan out blog post

### PR DESCRIPTION
**Pull request by nvkelso**

### CODE REVIEW

File: www/blog/2019/05/09/changes/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+decision to offer free hosting for projects and frankly we feel pretty bad that
+we have ended up being the project that has become a hassle for them.
+
+So, here's what's going to happen:
+
+* All the data files in the original `whosonfirst-data` repository have been
+  migrated in to per-country repositories. The naming convention for these
+  repositories is `whosonfirst-data-admin-{TWO_LETTER_WOF_COUNTRY_CODE}`.
+
+* Any data that isn't associated with a country will be migrated to the
```
Comment (nvkelso) [inline www/blog/2019/05/09/changes/index.md:56]: Resolve extra `` mark, breaks formatting

File: www/blog/2019/05/09/changes/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  ``whosonfirst-data-admin-xy` repository. This includes things like continents,
```
Comment (nvkelso) [inline www/blog/2019/05/09/changes/index.md:57]: Could link to Null Island in the Spelunker?

**Comment by thisisaaronland** [comment]
Huh, this merged in to master. Okay...

**Comment by nvkelso** [comment]
(I didn't make change commits only made comments on the PR.)

**Pull request merged**